solidity/contracts/helpers/TestLiquidityProtectionEventsSubscriber.sol

```diff
@@ -7,13 +7,17 @@ import "../liquidity-protection/interfaces/ILiquidityProtectionEventsSubscriber.
  * @dev Liquidity protection events subscriber interface
  */
 contract TestLiquidityProtectionEventsSubscriber is ILiquidityProtectionEventsSubscriber {
-    uint256 private _id;
-    address private _provider;
-    IConverterAnchor private _poolAnchor;
-    IReserveToken private _reserveToken;
-    uint256 private _poolAmount;
-    uint256 private _reserveAmount;
-    bool private _adding;
+    struct LiquidityProtectionEvent {
```

Comment:
👍

Reply:
I thought about it, but`LiquidityProtectionEvent` seems like the best name, since it describes exactly what it stores. I'll think about it a little bit more, but for the time being, let's keep it as is